test/utils/version/version.go

```diff
@@ -0,0 +1,29 @@
+package version
+
+import (
+	"context"
+	"fmt"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/operator-framework/api/pkg/lib/version"
+	operatorsv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+)
+
+var defaultNamespace = "numaresources-operator"
```

Comment:
Please pass the Namespace as an argument to the function so we'll be able to control it